sdk/storage/storage-blob/test/aborter.spec.ts

```diff
@@ -49,7 +48,6 @@ describe("Aborter", () => {
       assert.fail();
     } catch (err: any) {
       assert.equal(err.name, "AbortError");
-      assert.equal(err.message, "The operation was aborted.", "Unexpected error caught: " + err);
```

Comment:
Why removed this line? Is the message deleted or changed?

Reply:
The message is slightly different in browsers if I recall correctly, since it gets handled by `fetch` and thrown from there